core/web-client/src/App.vue

```diff
@@ -70,7 +70,19 @@ export default {
         .then(async (authenticated) => {
           if (authenticated) {
             const user = keycloak.getUser();
-            await this.$store.dispatch("login", user);
```

Comment:
@DaTebe 

the simple assumption is that the user does not exist any more in KC and be safely deleted from our storage. Any ideas where this could be an issue? 

One problem is that this is a hard delete, somewhat we don not practice currently. On the other hand there can not be 2 users with the same email in Mongodb. We have to make thought about Deletion in general